sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -1653,9 +1675,33 @@ def hpat_pandas_series_copy_impl(self, deep=True):
 @sdc_overload_method(SeriesType, 'corr')
 def hpat_pandas_series_corr(self, other, method='pearson', min_periods=None):
     """
-    Pandas Series method :meth:`pandas.Series.corr` implementation.
+    Intel Scalable Dataframe Compiler User Guide
+    ********************************************
 
-    Note: Unsupported mixed numeric and string data
+    Pandas API: pandas.Series.corr
```

Comment:
Please add limitations section and specify what only `method='pearson'` is supported
Also please add check for the method